pom.xml

```diff
@@ -16,8 +16,8 @@
     <properties>
         <java.version>21</java.version>
         <start-class>uk.gov.companieshouse.pscdataapi.PscDataApiApplication</start-class>
-        <spring.boot.version>3.3.5</spring.boot.version>
-        <spring-boot-maven-plugin.version>3.3.5</spring-boot-maven-plugin.version>
+        <spring-boot-dependencies.version>3.3.1</spring-boot-dependencies.version>
```

Comment:
Can this be 3.3.5 if 3.4.0 is breaking tests?

Reply:
done, 3.3.6 works so gone up to that